src/server_manager/install_scripts/install_server.sh

```diff
@@ -304,6 +304,7 @@ function start_shadowbox() {
   local -ar docker_shadowbox_flags=(
     --name "${CONTAINER_NAME}" --restart always --net host
     --label 'com.centurylinklabs.watchtower.enable=true'
+    --label 'com.centurylinklabs.watchtower.scope=shadowbox'
```

Comment:
Will this prevent watchtower itself from being updated?

Reply:
No, the label and the flag will be applied simultaneously. So old version without the label and the flag will update itself to a new version with the label and the flag.

Reply:
Changed scope name to make it a bit more explicit. PTAL